Formula/hadoop.rb

```diff
@@ -15,27 +15,51 @@ class Hadoop < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "92cb92589ac48035f2482eca9321cebe31dfd3873272da601e24f30cc758cb5e"
   end
 
-  depends_on "openjdk"
+  # WARNING: Check https://cwiki.apache.org/confluence/display/HADOOP/Hadoop+Java+Versions before updating JDK version
+  depends_on "openjdk@11"
 
   conflicts_with "yarn", because: "both install `yarn` binaries"
 
   def install
     rm_f Dir["bin/*.cmd", "sbin/*.cmd", "libexec/*.cmd", "etc/hadoop/*.cmd"]
     libexec.install %w[bin sbin libexec share etc]
     Dir["#{libexec}/bin/*"].each do |path|
```

Comment:
```suggestion
    (libexec/"bin").each_child do |path|
```
I know this is part of your patch, but may as well fix it while we're here.

Reply:
done